src/corefx/System.Globalization.Native/collation.cpp

```diff
@@ -0,0 +1,254 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#include <assert.h>
+#include <stdint.h>
+#include <unicode/uchar.h>
+#include <unicode/ucol.h>
+#include <unicode/usearch.h>
+#include <unicode/utf16.h>
+
+const int32_t CompareOptionsIgnoreCase = 1;
+// const int32_t CompareOptionsIgnoreNonSpace = 2;
+// const int32_t CompareOptionsIgnoreSymbols = 4;
+// const int32_t CompareOptionsIgnoreKanaType = 8;
+// const int32_t CompareOptionsIgnoreWidth = 0x10;
+// const int32_t CompareOptionsStringSort = 0x20000000;
```

Comment:
Yes, we don't currently support them but we will need to.

I would like to read the C++ guidelines that @nguerrera and @sokket drafted and then update the code in System.Globalization.Native to reflect it (including running the code formater).  I wrote some of this stuff before there were any guidelines.  I will do that next week, if it's okay with you?